Gemfile

```diff
@@ -1,4 +1,2 @@
 source 'https://rubygems.org'
 gemspec
-
-gem 'redis'
```

Comment:
I'd rather avoid using ENV vars for installation. That can make things more complicated for users, and libs upstream that use Classifier.